plain-ai/suggested-responses.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Suggested Responses'
+---
+Suggested responses are shown for incoming support requests. The responses use information from your [Knowledge Sources](/plain-ai/knowledge-sources) and information from the thread.
+
```

Comment:
Can we change this to say it uses past support requests as well as your documentation if you want?